### ISSUE TITLE
incomplete call can cause phantom note to appear

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#395]
```py
def g(a=1): pass  # type: ignore[no-untyped-def]

g(
)# type: ignore[no-untyped-call]
```
```
👉 mypy test.py
test.py:3:1: note: Type is "def (a: int =) -> Untyped"
Success: no issues found in 1 source file
```
